app/actions/remote/search.ts

```diff
@@ -134,5 +140,11 @@ export const searchPosts = async (serverUrl: string, params: PostSearchParams):
         return {error};
     }
 
-    return processPostsFetched(serverUrl, '', data, false);
+    const result = processPostsFetched(data);
+    await operator.handlePosts({
```

Comment:
why are we storing when previously we didn't ?

Reply:
As we are passing `fetchOnly` is  `false`, `processPostsFeched` is used to insert. As we removed that functionality, we are handling it here.